.gitignore

```diff
@@ -761,4 +761,6 @@ MigrationBackup/
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
 
-# End of https://www.toptal.com/developers/gitignore/api/windows,macos,linux,python,react,pycharm,emacs,vim,visualstudio,visualstudiocode
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/windows,macos,linux,python,react,pycharm,emacs,vim,visualstudio,visualstudiocode
+
+/docs/api/*.rst
```

Comment:
Need to remove, we shouldn't be ignoring anything for the api docs rst files now